testsuite/MDAnalysisTests/lib/test_qcprot.py

```diff
@@ -57,10 +57,20 @@ def test_regression(self, dtype):
 
         r = qcprot.CalcRMSDRotationalMatrix(p1, p2, len(prot), rot, weights)
 
-        rot_ref = np.array([0.999998, 0.001696, 0.001004,
-                            -0.001698,  0.999998,  0.001373,
-                            -0.001002, -0.001375,  0.999999],
-                           dtype=dtype)
+        rot_ref = np.array(
+            [
+                0.999998,
+                0.001696,
+                0.001004,
+                -0.001698,
+                0.999998,
+                0.001373,
+                -0.001002,
+                -0.001375,
+                0.999999,
+            ],
+            dtype=dtype,
+        )
```

Comment:
```suggestion
        rot_ref = np.array(
            [
                [0.999998,
                0.001696,
                0.001004],
                [-0.001698,
                0.999998,
                0.001373],
                [-0.001002,
                -0.001375,
                0.999999],
            ],
            dtype=dtype,
        ).flatten()
```

Reply:
I feel the change of the shape and the use of `flatten()` are too far from "just formatting changes". I'd say to turn off formatting if we want to keep the matrix-like one, and eventually revisit on a separate PR. Or simply keep the new foamtting.

Reply:
Agreed on "more than formatting" - in this case there is absolutely no "worth" in the formatting, so either option works. That being said, if we're actively trying to find ways to make the new version better, then I'm interpreting this as a bias towards the old version.

Reply:
Since the output of the function is indeed a 1D array, I'll keep the new notation for consistency (otherwise we are starting to introduce arbitrary formatting for 2D/1D data which might be difficult to track). We can eventually reconsider in the future if needed and more cases like this pop up in other PRs. (I'm taking your approval as I would prefer this, but it's not major and therefore fine either way).